package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.0.7"
+    "nan": "^2.0.7"
```

Comment:
@defunctzombie Finally found the thread where we discussed this.

*Note to all:* libxmljs is now using a tilde (`~`) before the NAN version in package.json.

Yes, @defunctzombie, this directly goes against what we both agreed, but a tilde will allow newer patch versions which I believe is okay since NAN *does* follow semver.

I think that allowing patch versions of NAN is important because of the way NAN header files interact with the differing versions of node-gyp that people have installed.

If a user has the latest node-gyp, and that introduces an issue with the NAN header files, NAN will release a patch and our lib will be fixed automatically.

Ideally, it will save us headache in the long run. Especially since patches usually get out faster for NAN than they do for libxmljs.